lib/themes/dosomething/paraneue_dosomething/scss/patterns/_card.scss

```diff
@@ -66,6 +66,17 @@
   .form-actions {
     padding-top: gutter();
   }
+
+  &.-signup {
+    width: 30%;
+    min-width: 256px;
+    margin: 1.5%;
+    display: inline-block;
+
+    .beta-card__row:first-of-type {
+      margin-top: 0;
+    }
+  }
```

Comment:
For example, this is what it looks like if we removed the margin from .beta_card__row

<img width="310" alt="screen shot 2016-09-20 at 12 30 12 pm" src="https://cloud.githubusercontent.com/assets/897368/18679528/01fcae9a-7f2e-11e6-8cb2-183357b9d0fa.png">

If I remove the `.-headerless`, you can see where `.beta-card__row` is applying the margin

<img width="360" alt="screen shot 2016-09-20 at 12 31 17 pm" src="https://cloud.githubusercontent.com/assets/897368/18679575/3f3b48ca-7f2e-11e6-8478-d5be556920d8.png">